Improved Contributors Section

Vidip-Ghosh opened the pull request:
<!-- Pull Request Template -->

## Related Issue

Closes #599 

<!-- If there is no issue number, the PR will not be merged. Therefore, please ensure that the issue number is added -->

## Description

I have added a new color scheme to the contributors section of the website. 

## Screenshots

Before:

<img width="1661" alt="Screenshot 2023-06-13 at 22 43 41" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/444b2c89-93fd-43f0-827b-512e70b6b755">


After:

<img width="1680" alt="Screenshot 2023-06-13 at 22 40 27" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/dd9e017c-b90f-4ec9-8e9d-29907c10b444">


## Checklist 

<!-- [x] - To mark checked, put 'x' in place of ' '(space)  -->
<!-- [ ] - Keep unchecked using ' '(space)  -->

- [x] My code adheres to the established style guidelines of the project.
- [ ] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

JasonDsouza212 commented:
@Vidip-Ghosh, I believe you may be confused. If you click on the open-source community, you will be able to see all the people who have contributed. Therefore, I would like that page to be redesigned, not the one you have edited

JasonDsouza212 commented:
![Screenshot_20230613-233030](https://github.com/JasonDsouza212/free-hit/assets/74404894/b1a6e1cc-ab81-4a4d-8d39-883798f0bb3d)

Vidip-Ghosh commented:
> @Vidip-Ghosh, I believe you may be confused. If you click on the open-source community, you will be able to see all the people who have contributed. Therefore, I would like that page to be redesigned, not the one you have edited

Okay I got it

Vidip-Ghosh commented:
I have changed the color scheme. I changed into opacity from grayscale. 

**Before**
<img width="1680" alt="Screenshot 2023-06-14 at 19 48 12" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/f3763265-a522-409b-8a62-23952306ba0d">

**After**
<img width="1680" alt="Screenshot 2023-06-14 at 19 47 36" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/fd63763d-3643-422e-a8c0-f19f84f0138d">

Please check if its alright.

JasonDsouza212 commented:
@Vidip-Ghosh, please try to maintain the same theme. The suggestion I can give is that you display the username in a card, but in a more organized and attractive manner.  make sure to explore all the data that the API call is returning and implement the necessary data, keeping it concise and tidy.

JasonDsouza212 commented:
@Vidip-Ghosh, will you be working on this, or should I assign it to someone else?

Vidip-Ghosh commented:
> @Vidip-Ghosh, will you be working on this, or should I assign it to someone else?

Yes I will be working

VenomFate-619 commented:
@Vidip-Ghosh post the ss of the changes

Vidip-Ghosh commented:
> @Vidip-Ghosh post the ss of the changes

<img width="1680" alt="Screenshot 2023-06-21 at 16 54 11" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/0c95746f-fa2c-403c-a10a-a12f100dd9df">
<img width="1680" alt="Screenshot 2023-06-21 at 16 53 36" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/88a347c5-42f1-434e-955f-43f06ff91cdd">

JasonDsouza212 commented:
Dont add any color that is out of the theme @Vidip-Ghosh

JasonDsouza212 commented:
The contribution page looks the same as before. Let's try something new and attractive that aligns with our theme.

Vidip-Ghosh commented:
> The contribution page looks the same as before. Let's try something new and attractive that aligns with our theme.

Can you please suggest some ideas which can be done? Like I added name of the contributors. Any other ideas

JasonDsouza212 commented:
You can create a small card where you can add a title. It should also be responsive, and you can include the necessary information. Just make sure to maintain the color theme and avoid using other colors.

VenomFate-619 commented on src/pages/Community.jsx:
```diff
@@ -38,19 +38,23 @@
-                  </a>
+                  <div className='contributor-card'>
+                    <a
+                      className="contributor-anchor"
```
fix this.
add this line ``` rel="noopener noreferrer"```

Vidip-Ghosh replied:
Yes I have solved

JasonDsouza212 commented:
![Screenshot from 2023-06-23 20-59-25](https://github.com/JasonDsouza212/free-hit/assets/74404894/1522b0d6-4d3c-4754-8a13-70e9baa87ed4)
![Screenshot from 2023-06-23 20-59-07](https://github.com/JasonDsouza212/free-hit/assets/74404894/dbfb5f5a-ef48-4edf-8b74-ecc8024c7bcf)

The Navbar has style issues, and the cards have a pure white background. Additionally, the title of contributions is thin and looks odd. You can make the contribution page card text font size a little larger. Similarly, the name should be larger so that it matches the size of the commits, and the border should be slightly bigger as well.

Vidip-Ghosh commented:
<img width="1680" alt="Screenshot 2023-06-24 at 18 37 19" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/b0565d2b-bb29-4f62-ba10-9434a0a5ebdf">
Made font size of contribution page card text a little larger

Vidip-Ghosh commented:
@JasonDsouza212 @VenomFate-619 Please review this PR

JasonDsouza212 commented:
@Vidip-Ghosh yes ,I will

JasonDsouza212 commented:
The changes made to the contributor section design were not planned or necessary. Please revert them back to the original design as it is in the production application.
![Screenshot from 2023-06-30 20-00-07](https://github.com/JasonDsouza212/free-hit/assets/74404894/b09179e6-fbb2-4112-84c2-ce7ac8cb0c6d)

As you can see, the header is not properly aligned. Additionally, the background of the cards is white, which makes it look improper. Therefore, please set the background of the cards to match the overall white background. Apart from these issues, everything looks good.
![Screenshot from 2023-06-30 19-59-46](https://github.com/JasonDsouza212/free-hit/assets/74404894/2b08a053-16fd-46ba-a234-e2ad43974d7e)

Vidip-Ghosh commented:
> The changes made to the contributor section design were not planned or necessary. Please revert them back to the original design as it is in the production application. ![Screenshot from 2023-06-30 20-00-07](https://user-images.githubusercontent.com/74404894/250116287-b09179e6-fbb2-4112-84c2-ce7ac8cb0c6d.png)
> 
> As you can see, the header is not properly aligned. Additionally, the background of the cards is white, which makes it look improper. Therefore, please set the background of the cards to match the overall white background. Apart from these issues, everything looks good. ![Screenshot from 2023-06-30 19-59-46](https://user-images.githubusercontent.com/74404894/250116298-2b08a053-16fd-46ba-a234-e2ad43974d7e.png)

Okay alright

Vidip-Ghosh commented:
Now check @JasonDsouza212 
<img width="1680" alt="Screenshot 2023-06-30 at 22 28 26" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/c02c8844-1c3b-42e7-8be5-fccac5c7eed9">
<img width="527" alt="Screenshot 2023-06-30 at 22 27 39" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/28b681e9-8e1b-42d2-9c1f-51749920e0a4">
<img width="471" alt="Screenshot 2023-06-30 at 22 27 21" src="https://github.com/JasonDsouza212/free-hit/assets/91741581/54a15a33-eb42-43fe-b1f2-11949a13fe5a">

Vidip-Ghosh commented:
@JasonDsouza212 @VenomFate-619 Why its showing the label **title-needs-formatting**?

JasonDsouza212 commented:
Hello, contributors! This is Jason speaking. Due to some workflow issues, we will be closing all the PRs today. Therefore, I kindly request all of you to create new PRs tomorrow, as the currently open ones will be closed by our team. I apologize for any inconvenience caused. Also, please make sure to check the titles when creating the PRs. You can refer to the README.md for more details. Thank you.

JasonDsouza212 commented:
@Vidip-Ghosh, please create a new PR. I believe everything looks good.
